reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -580,6 +586,59 @@ def apply_wiener_filter(projection, ctf, small_number):
         projection_wfilter_f = projection * wfilter
         return projection_wfilter_f
 
+    @staticmethod
+    def compute_ssnr(projections, ctfs_list, small_number=0.01):
+        """Compute spectral signal to noise ratio (SSNR) for each pixel of projections.
+
+        Uses section 2.6:
+        https://doi.org/10.1016/j.jsb.2011.06.010
+
+        Parameters
+        ----------
+        projections : arr
+            Shape (n_projections, n_pix, n_pix)
+        ctfs_list : list
```

Comment:
refactor so ctfs are np.ndarray and do not have to convert them from list to array